pkg/engine2/testdata/k8s_api.err.json

```diff
@@ -0,0 +1,117 @@
+[
+  {
+    "error": {
+      "chain": [
+        "required property Id is not set on resource aws:subnet:vpc-0:subnet-0"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "Id",
+    "resource": "aws:subnet:vpc-0:subnet-0",
+    "value": null
+  },
+  {
+    "error": {
+      "chain": [
+        "required property Id is not set on resource aws:subnet:vpc-0:subnet-1"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "Id",
+    "resource": "aws:subnet:vpc-0:subnet-1",
+    "value": null
+  },
+  {
+    "error": {
+      "chain": [
+        "required property Id is not set on resource aws:subnet:vpc-0:subnet-2"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "Id",
+    "resource": "aws:subnet:vpc-0:subnet-2",
+    "value": null
+  },
+  {
+    "error": {
+      "chain": [
+        "required property Id is not set on resource aws:subnet:vpc-0:subnet-3"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "Id",
+    "resource": "aws:subnet:vpc-0:subnet-3",
+    "value": null
+  },
+  {
+    "error": {
+      "chain": [
+        "required property Arn is not set on resource aws:vpc:vpc-0"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "Arn",
+    "resource": "aws:vpc:vpc-0",
+    "value": null
+  },
+  {
+    "error": {
+      "chain": [
+        "required property BaseImage is not set on resource aws:ecr_image:pod2-ecr_image"
+      ]
+    },
+    "error_code": "config_invalid",
+    "property": "BaseImage",
+    "resource": "aws:ecr_image:pod2-ecr_image",
+    "value": null
+  },
+  {
+    "error": {
+      "children": [
+        {
+          "chain": [
+            "invalid int value 80"
```

Comment:
I think this is because they're `"80" (string)` and not being converted to an `int`

Reply:
that should be able to be converted but if not we need to either cut a card or update the code